code/__DEFINES/MC.dm

```diff
@@ -57,6 +57,9 @@
 /// This flag overrides SS_KEEP_TIMING
 #define SS_POST_FIRE_TIMING 64
 
+/** Subsystem is still initialized in the lightweight sleeper-server mode */
+#define SS_SLEEPER 128
```

Comment:
This name seems confusing to me, sleeper sounds like it would sleep if it was a sleeper server